test/html.js

```diff
@@ -277,11 +277,6 @@ describe('html', function() {
       )
     );
 
-    // minifyJson
```

Comment:
Because htmlnano is no longer responsible for JSON minifying, although this caused the tests to not show that it actually changed json into JS so I've put it back